release/ray_release/test.py

```diff
@@ -4,7 +4,7 @@
 DEFAULT_PYTHON_VERSION = tuple(
     int(v) for v in os.environ.get("RELEASE_PY", "3.7").split(".")
 )
-DOCKER_REPO = "029272617770.dkr.ecr.us-west-2.amazonaws.com/anyscale"
+DATAPLANE_ECR_REPO = "029272617770.dkr.ecr.us-west-2.amazonaws.com/anyscale"
```

Comment:
maybe still keep the images in two repos for consistency? one is `anyscale/ray` and one is `anyscale/ray-ml` ?